systemd/tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py37
+envlist =
+    {py27,py37}-{systemd}
+    flake8
+
+[testenv]
+usedevelop = true
+platform = linux|darwin|win32
+deps =
+    -e../datadog_checks_base[deps]
+    -rrequirements-dev.txt
+passenv =
+    DOCKER*
+    COMPOSE*
+
+commands =
+    pip install -r requirements.in
+    pytest -v
+
+[testenv:flake8]
```

Comment:
@therve Do we need this now?

Reply:
It looks like it doesn't have dd_style_check = True, so yes. We'll need to update those later on, so it can definitely stay here for now.